File: Python/String-1/left2.py
"""
Dada uma String "str", retorne outra em que as 2 primeiras letras sejam reposicionadas
para o final. A string dada terá tamanho de pelo menos 2.
"""


def left2(str):
    return str[2:] + str[:2]


print(left2("lucas"))

File: String-1/src/left2/Left2.java
package left2;

public class Left2 {
    //Retorne a String dada de modo que suas primeiras duas letras estejam no fim. A String terá que ter pelo menos 2 de
    //length. (Ex.: ("CASA") -> "SACA").

    public String left2(String str) {
        return str.substring(2) + str.substring(0,2);
    }
}
